package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@journeyapps/pdf-reports",
-  "version": "2.0.0",
+  "version": "1.0.0",
```

Comment:
Is this change correct?

Reply:
Yes, the changeset apply will bump it from `1.0.0` to `2.0.0` again. Latest version on `npm` is still `1.0.0` atm
https://www.npmjs.com/package/@journeyapps/pdf-reports